### PR TITLE
Migrate ListPageBody to the SDK

### PR DESCRIPTION
**Pull request by fhlavac**
Migrated ListPageBody component to the SDK as a part of ListPage migration effort
https://issues.redhat.com/browse/HAC-311

@invincibleJai @andrewballantyne @florkbr

### CODE REVIEW

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/index.ts [hunk 1]
```diff
@@ -7,3 +7,4 @@ export { default as ResourceStatus } from './utils/resource-status';
 
 export * from './status/icons';
 export * from './status/statuses';
+export { default as ListPageBody } from './factory/ListPage/ListPageBody';
```
Comment (florkbr) [inline frontend/packages/console-dynamic-plugin-sdk/src/app/components/index.ts:10]: @fhlavac I know this wasn't caught by the linter, but can we put this export above the wildcard `*` exports?

File: frontend/packages/console-dynamic-plugin-sdk/src/styles/common.scss [hunk 2]
```diff
@@ -0,0 +1,13 @@
+@import './vars.scss';
+
+.dps-m-pane__body {
```
Comment (florkbr) [inline frontend/packages/console-dynamic-plugin-sdk/src/styles/common.scss:3]: @fhlavac how did you derive these styles? Did they come from the console?

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import * as React from 'react';
+import '../../../../styles/common.scss';
```
Comment (florkbr) [inline frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx:2]: Not blocking: at some point we should investigate using aliases so we don't need to use deep relative paths such as `../../../../`

File: frontend/packages/console-dynamic-plugin-sdk/src/styles/common.scss (hunk 2 above)
Comment (fhlavac) [reply frontend/packages/console-dynamic-plugin-sdk/src/styles/common.scss:3]: @florkbr yep, they are in common and present when the console loads. Just thought we want to separate them to the SDK. Is it ok to remain the component dependent on the common styles?

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx (hunk 3 above)
Comment (fhlavac) [reply frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx]: used the alias, thank you :-)

File: frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 
-export const ListPageBody: React.FC = ({ children }) => {
+const ListPageBody: React.FC = ({ children }) => {
```
Comment (andrewballantyne) [inline frontend/packages/console-dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody.tsx:3]: Weird -- I wonder why it was done this way.

File: frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts [hunk 5]
```diff
@@ -62,8 +62,7 @@ export const ListPageCreateButton: React.FC<ListPageCreateButtonProps> = require
   .ListPageCreateButton;
 export const ListPageCreateDropdown: React.FC<ListPageCreateDropdownProps> = require('@console/internal/components/factory/ListPage/ListPageCreate')
   .ListPageCreateDropdown;
-export const ListPageBody: React.FC = require('@console/internal/components/factory/ListPage/ListPageBody')
-  .default;
+export { default as ListPageBody } from '@console/dynamic-plugin-sdk/src/app/components/factory/ListPage/ListPageBody';
```
Comment (andrewballantyne) [inline frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts:65]: Is this needed with the change to `frontend/packages/console-dynamic-plugin-sdk/src/app/components/index.ts`?

Comment (fhlavac) [reply frontend/packages/console-dynamic-plugin-sdk/src/api/dynamic-core-api.ts]: removed, thank you :slightly_smiling_face:

**Review by andrewballantyne — APPROVED** [review]
Pretty smooth migration. Looks good to me.

Migration does not need documentation.
/label docs-approved
/label px-approved

cc @florkbr please do a final review here

**Review by andrewballantyne — APPROVED** [review]
/lgtm

Simple move, I've tested the various locations where tables are used and everything looks good. 
/label qe-approved

**Pull request merged**